ipfs/ipfs.go

```diff
@@ -0,0 +1,72 @@
+package ipfs
```

Comment:
Is there a reason why you haven't used already written client like https://github.com/ipfs/go-ipfs-api but you go with own implementation? Does this client have some limitation that stops us from using it?

Reply:
hum ok, ill check this

Reply:
this can be an improvement later